enclave-tls/src/include/enclave-tls/log.h

```diff
@@ -7,12 +7,16 @@
 #ifndef _ENCLAVE_LOG_H_
 #define _ENCLAVE_LOG_H_
 
+// clang-format off
 #include <time.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <enclave-tls/api.h>
+#ifdef SGX
+#include <enclave-tls/sgx.h>
+#endif
 
-// clang-format off
+#ifndef SGX
 #ifdef OCCLUM
```

Comment:
change to `#if defined(SGX) || defined(OCCLUM)`

Reply:
not SGX